src/applications/lgy/coe/config/chapters/loans/loanHistory.js

```diff
@@ -35,25 +38,55 @@ export const uiSchema = {
     'ui:options': {
       itemName: 'VA-backed Loan',
       viewField: PreviousLoanView,
+      keepInPageOnReview: true,
     },
     items: {
       'ui:title': 'Previous loan information',
       'ui:options': {
         itemName: 'VA-backed loan',
       },
-      dateRange: dateRangeUI(
-        'Date your loan began',
-        'Date you paid off your loan (Leave this blank if it’s not paid off)',
-        'Date loan ended must be after the start of the loan',
-      ),
+      dateRange: {
```

Comment:
Was originally using the platform provided `dateRangeUi`, but needed the ability to include `ui:options` easily, hence the unabbreviated version of it.